ES/ESmr039.xml

```diff
@@ -126,7 +126,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </list>
                         <seg type="rubrication">Nomina sacra; incipit of Text I; directives to the
```

Comment:
Did we not say "Nomina sacra" is the wrong term?

Reply:
Yes, we did.